pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -89,7 +89,8 @@
     public void getActiveBrokers(@Suspended final AsyncResponse asyncResponse,
                                  @PathParam("cluster") String cluster) {
         validateSuperUserAccessAsync()
-                .thenCompose(__ -> validateClusterOwnershipAsync(cluster))
+                .thenCompose(__ -> cluster == null ? CompletableFuture.completedFuture(null)
+                        : validateClusterOwnershipAsync(cluster))
```

Comment:
No need to modify this line.

Reply:
Yes, `validateClusterOwnershipAsync` could also pass when cluster=null